include/dlpack/dlpack.h

```diff
@@ -68,6 +68,9 @@ typedef enum {
    * \brief CUDA managed/unified memory allocated by cudaMallocManaged
    */
   kDLCUDAManaged = 13,
+  kDLONEAPI_GPU = 14,
+  kDLONEAPI_CPU = 15,
+  kDLONEAPI_ACCELERATOR = 16,
```

Comment:
Could you add short comments for each entry for better clarity?

Reply:
Addressed.